packages/taraxa-ui/src/components/DataCard/DataCard.styles.ts

```diff
@@ -9,7 +9,7 @@ const useStyles = makeStyles(() => {
       backgroundColor: '#151823',
       paddingLeft: '24px',
       paddingRight: '24px',
-      border: '1px solid #15AC5B',
+      border: '1px solid #15AC5B !important',
```

Comment:
Why do we need important here? Does it get overwritten somewhere else?

Reply:
Yes, it gets overridden for some reason by the `Card` component `root` styles.